src/components/CertificateTemplates/tlds/sg/edu/nus/NUS-K3MBBS-2019/integration.spec.js

```diff
@@ -0,0 +1,82 @@
+import { Selector } from "testcafe";
+import { readFileSync } from "fs";
+import { join } from "path";
+import { getData } from "@govtechsg/open-attestation";
+
+fixture("National University of Singapore").page`http://localhost:3000`;
+
+const Certificate = "./sample.opencert";
+
+const RenderedCertificate = Selector("#rendered-certificate");
+
+const validateTextContent = async (t, component, texts) =>
+  texts.reduce(
+    async (prev, curr) => t.expect(component.textContent).contains(curr),
+    Promise.resolve()
+  );
+
+test("NUS-K3MBBS-2019 certificate is rendered correctly", async t => {
+  // Inject javascript and execute window.opencerts.renderDocument
+  const certificateContent = getData(
+    JSON.parse(readFileSync(join(__dirname, Certificate)).toString())
+  );
+  await t.eval(() => window.opencerts.renderDocument(certificateContent), {
+    dependencies: { certificateContent }
+  });
+
+  // Check content of window.opencerts.templates
+  await t.wait(500);
```

Comment:
could you replace t.wait(500) with selector instead? we realised that t.wait(500) was causing flaky test failures
refer to https://github.com/OpenCerts/legacy-templates/commit/caf3f99e435cfaad22406b393a2fa6a5c7690894

Reply:
yes, replaced.